Frontend/src/components/pages/partial/SearchBar.jsx

```diff
@@ -4,6 +4,10 @@ import { Search as SearchIcon } from "@mui/icons-material";
 export default function SearchBar({ type, data, setSearchResults }) {
   function handleChange(value) {
     value = value.toLowerCase();
+    if (!data) {
```

Comment:
ovo bio neki bug ili? 
gledaj da backend uvijek vraca prazan array ako nema podataka o necemu tkd ne mozes primiti null od backenda

takoder jel postoji situacija u kojoj ce se searchbar skrsit s obzirom da u retku 25 imamo return true kojim mislim da handle-a ovo

Reply:
Dap, to je bio bug, da se kršilo kada je collaborations bio prazan. Ovo ga je ispravilo.

Reply:
Nisam uspio ponoviti bug koji je ovo ispravilo, moguće je da ga je nešto drugo što sam napravio maknulo. Izbrisao taj kod.